hunt/app/views/animals/index.html.erb

```diff
@@ -1,18 +1,23 @@
 <ul>
```

Comment:
We can delete these opening and closing `<ul>` tags. Doesn't look like they're used.